net/dawn/Makefile

```diff
@@ -5,13 +5,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dawn
-PKG_SOURCE_DATE:=2020-03-23
-PKG_RELEASE:=1
+PKG_SOURCE_DATE:=2020-03-28
+PKG_RELEASE:=2
```

Comment:
Why?

Reply:
Because I updated the source?

Reply:
https://github.com/berlin-open-wireless-lab/DAWN/commit/b967c1e0b68bc05ec5456ab4ecb1a7f0ae75a1b7

Reply:
No. Please read [CONTRIBUTING.md](https://github.com/openwrt/packages/blame/master/CONTRIBUTING.md#L36).

Reply:
But updaing PKG_SOURCE_DATE is correct or?

Reply:
Yes, updating PKG_SOURCE_DATE is ok.